data/containers/app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask, render_template
```

Comment:
Since not all people know Flask, we might want to add a small explanatory header about the function, something like

```suggestion
#!/usr/bin/env python3
##
## Simple Flask test program (see https://flask.palletsprojects.com/en/2.0.x/quickstart/)
## Serve 'index.html' from the `templates` directory
##
## Usage: python3 app.py
## Bind to port 80 or to the PORT environment variable, if set

from flask import Flask, render_template
```

Reply:
ready for review